cmd/grpcurl/grpcurl.go

```diff
@@ -313,6 +319,13 @@ func main() {
 		return cc
 	}
 
+	var headers []string
+	if *expandHeaders {
+		headers = grpcurl.ExpandHeaders(addlHeaders)
```

Comment:
Also need to expand `rpcHeaders` and `reflHeaders`.

Reply:
Done.